### DISCUSSION TITLE
TS101

### DISCUSSION THREAD

**Discussion by ykhnhfr** [Ralim/IronOS#1447] [Q&A]
Hi, 

is the Firmware also compatible with the TS101 Version which is upgraded TS100?

https://www.miniware.com.cn/product/ts101-smart-soldering-iron/

Tried to flash but it didn't work yet. 

Regards

Yannick

**Comment by Ralim** [comment]
This is in progress over here https://github.com/Ralim/IronOS/issues/1420.
At this point its not a priority to me as the hardware is not open; and they have made some frustrating design choices so motivation is low.

**Reply by MaximusPanin** [reply]
Hello. Can I get details ? And then suddenly you have to sell your TS101 and return to TS100 ))
